frontend/src/components/checkout/AddressForm.jsx

```diff
@@ -33,10 +33,14 @@ const AddressForm = ({
       address.address.line1.length > 100 ||
       address.address.line2?.length > 100 ||
       address.address.city.length > 100 ||
-      address.address.postal_code.length > 20 ||
       address.name.length > 100
     ) {
-      setErrorMessage("the fields must be less than 100 characters and corrects");
+      setErrorMessage("The name,city and lines fields cannot be longer than 100 characters");
```

Comment:
It's misleading.
Is it possible to have  validation message for each field?

Otherwise how about this message "Invalid fields with more than 100 characters"

Reply:
Okay, I will take your suggestion

Reply:
Done